march_state_machine/src/march_state_machine/healthy_sm.py

```diff
@@ -109,6 +110,10 @@ def create():
                                transitions={'succeeded': 'STANDING',
                                             'preempted': 'failed', 'failed': 'UNKNOWN'})
 
+        smach.StateMachine.add('GAIT RD SLOPE UP', ramp_door_slope_up_sm.create(), # RD stands for Ramp and Door
```

Comment:
This will fix your build
```suggestion
        smach.StateMachine.add('GAIT RD SLOPE UP', ramp_door_slope_up_sm.create(),  # RD stands for Ramp and Door
```